components/Banner/examples/supplemental/MDCBannerViewLayout.h

```diff
@@ -15,18 +15,24 @@
 #import <CoreGraphics/CoreGraphics.h>
 #import <Foundation/Foundation.h>
 
-@interface MDCBannerViewLayout : NSObject
+typedef NS_ENUM(NSInteger, MDCBannerViewLayoutStyle) {
+  MDCBannerViewLayoutStyleInvalid = 0,                 // Invalid style
```

Comment:
Is there a way we can remove `Invalid`, and somehow recalculate when needed?

Reply:
I have same feeling as you have on this state. But I still want to handle the boundary case using this enum. Since this is an internal class, I think it won't lead to confusion or usability issue if it sounds good to you?